.circleci/config.yml

```diff
@@ -491,6 +491,56 @@ jobs:
           name: Unit tests
           command: yarn --cwd packages/protocol test common/
 
+  protocol-verify-bytecodes:
+    <<: *defaults
+    resource_class: large
+    steps:
+      - attach_workspace:
+          at: ~/app
+      - run:
+          name: Check if the test should run
+          command: |
+            ./scripts/ci_check_if_test_should_run_v2.sh @celo/protocol
+      - run:
+          name: Verify bytecodes between RC1 and current branch
+          command: |
+            cd packages/protocol
+
+            # deploy RC1 contracts to ganache devchain
+            # We fetch via HTTPS instead of SSH to avoid the interactive "do you
+            # trust the RSA key fingerprint" prompt.
+            git remote add origin-https https://github.com/celo-org/celo-monorepo.git
+            git fetch origin-https rc1
```

Comment:
At this point, this is incorrect, as `master` needs to compare against release 1.

I believe there are only 3 options:

1. Adjust the migrations to proxy linked libraries and then checkout the release-1 branch here
2. Create a special case for release 2, where creating the bytecode involves 
  2.1 Deploying RC1
  2.2 Creating the proposal for release 1
  2.3 Get through the proposal for release 1
  2.4 Continue
3. Not add this to CI until 1. is done

Reply:
went with #3